WalletWasabi/WabiSabi/Models/RoundState.cs

```diff
@@ -9,29 +9,39 @@
 namespace WalletWasabi.WabiSabi.Models
 {
 	public record RoundState(
-		uint256 Id,
 		uint256? BlameOf,
 		CredentialIssuerParameters AmountCredentialIssuerParameters,
 		CredentialIssuerParameters VsizeCredentialIssuerParameters,
 		FeeRate FeeRate,
 		Phase Phase,
 		bool WasTransactionBroadcast,
+		TimeSpan InputRegistrationTimeout,
 		TimeSpan ConnectionConfirmationTimeout,
+		TimeSpan OutputRegistrationTimeout,
+		TimeSpan TransactionSigningTimeout,
+		long MinRegistrableAmount,
```

Comment:
`MinRegistrableAmount` is only used to populate the default value for `Allowed{In,Out}putAmount.Min`, so it is not needed on the client.

```suggestion
```

Reply:
I already used the range so, i need it on the client.